airflow/models/all_models/__init__.py

```diff
@@ -0,0 +1,154 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# flake8: noqa: F401
+# pylint: disable=redefined-outer-name, too-many-locals, too-many-return-statements, too-many-branches
+
+import sys
+
+__all__ = ['ID_LEN', 'Base', 'BaseJob', 'BaseOperator', 'BaseOperatorLink', 'Connection',
+           'DAG', 'DagModel', 'DagTag', 'DagBag', 'DagPickle', 'DagRun',
+           'ImportError', 'KubeResourceVersion', 'KubeWorkerIdentifier', 'Log',
+           'Pool', 'SerializedDagModel', 'SkipMixin', 'SlaMiss', 'TaskFail', 'TaskInstance',
+           'clear_task_instances', 'TaskReschedule', 'Variable', 'XCOM_RETURN_KEY',
+           'XCom']
+
+PY37 = sys.version_info >= (3, 7)
+
+def __getattr__(name):
```

Comment:
Here we do not want lazy loading in fact. The whole reason for having the "all_models" is to actually import those models - so it's ok to have just usual imports here.